raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -168,12 +179,12 @@ Builder withMemoryPool(MemoryPool pool) {
             return this;
         }
 
-        Builder withAppendLingerMs(int appendLingerMs) {
+        public Builder withAppendLingerMs(int appendLingerMs) {
```

Comment:
Mark these methods as `public` to be reused in `SnapshotWriterTest`, in `SnapshotWriterTest`, it need a way to advance the high watermark of the snapshot validation won't be passed